client/session_bootstrap_blocks_test.go

```diff
@@ -1392,11 +1392,18 @@ func (s *synchronousWorkerPool) Init() {
 	// Noop
 }
 
-func (s *synchronousWorkerPool) Go(work pool.Work) {
+func (s *synchronousWorkerPool) Go(work xsync.Work) {
 	work()
 }
 
-func (s *synchronousWorkerPool) GoIfAvailable(work pool.Work) bool {
+func (s *synchronousWorkerPool) GoIfAvailable(work xsync.Work) bool {
+	work()
+	return true
+}
+
+func (s *synchronousWorkerPool) GoWithTimeout(
```

Comment:
Oh, sorry, I misunderstood the question – yeah it's a new function for this Mock because we had two WorkerPools: one in M3DB and one in M3X. I dropped the one that has less functionality, so this function was added to match the interface.